lib/dogstatsd.rb

```diff
@@ -340,6 +372,9 @@ def send_stats(stat, delta, type, opts={})
     if sample_rate == 1 or rand < sample_rate
       # Replace Ruby module scoping with '.' and reserved chars (: | @) with underscores.
       stat = stat.to_s.gsub('::', '.').tr(':|@', '_')
+      if opts[:manual_sample_rate]
+        sample_rate *= opts[:manual_sample_rate]
```

Comment:
I kinda feel like this should transform into `reported_sample_rate`, maybe?

so as to not conflate the sample rate dogstatsd should be using with the sample rate we should be reporting.